### DISCUSSION TITLE
personal-developement-bahman-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#33] [Announcements]
# personal-developement-bahman-1402/

به سبک چند وقت گذشته، گزارش توسعه فردی ماهانه‌ام رو منتشر می‌کنم.
متمم تو متمم بیشتر درس‌های مسیر شغلی رو دنبال کردم. این مطالب علی‌الخصوص الان که در حال تغییر کار هستم برام مهم خواهد بود؛ چون به احتمال بسیار زیاد شغل بعدیم تا حد زیادی با کار الانم متفاوت خواهد بود (حتی اگر پوزیشن شغلی‌ام همچنان ML Engineer باشه). این ذات کارکردن تو حوزه هوشه و چندان نمیشه کاریش کرد.

https://aprd.ir/personal-developement-bahman-1402/

**Comment by iamamohsen** [comment]
سلام امیر جان امیدوارم که حالت خوب باشه
در پروفایل متمم مشغول بالا و پایین شدن بودم که به اسمت برخوردم و بعد به واسطه اون به این وبلاگ رسیدم.
وقتی که تعدادی از نوشته هات رو خوندم، بیشتر مشتاق شدم به خوندن از تو و این روزمرگی ها. اینکه ردپای صحبت ها و مدل ذهنی محمدرضا شعبانعلی رو هم در نوشته هات پیدا میکردم و میدیدم که چه دغدغه های مشابهی داریم، من رو بیشتر خوشحال میکرد. امیدوارم این کامنت آغازی باشه برای آشنایی بیشتر ما. من حالا در کنار وبلاگ شخصیت، کانال تلگرامت رو هم دنبال میکنم.

من هم مثل تو گاهی نوشته هایی از روزمرگی ها و چیزهایی که یاد میگیرم منتشر میکنم. اما بخاطر اینکه برنامه نویسی بلد نیستم و فرصتش هیچوقت برام پیش نیومد که جدی یادش بگیرم نتونستم وبلاگی رو بالا بیارم، با اینکه حتی یک بار دامین هم تهیه کردم. برای همین تصمیم گرفتم بیشتر در یک کانال تلگرام و ویرگول (که مثل تو دل خوشی ازش ندارم) نوشته هام رو منتشر کنم. اتفاقا من خیلی به دنبال کسی بودم که بتونم با کمکش وبلاگ شخصیم رو راه اندازی کنم، چراکه هیچ اطلاعاتی از برنامه نویسی و وردپرس و ... نداشتم. تا اینکه متوجه شدم که تو با Hugo این وبلاگ رو، (با طراحی جالب و مینیمالیش) بالا اوردی و بعد آموزش تو رو در آپارات دیدم. البته که مقداری تخصصی بود. حالا خواستم اینجا هم ازت بپرسم که آیا برای من که آموزش خاصی هم ندیدم Hugo رو پیشنهاد میکنی برای بالا اوردن یک وبلاگ معمولی یا پیشنهاد های بهتری داری؟

**Reply by pourmand1376** [reply]
سلام محسن جان. امیدوارم حال تو هم خیلی خوب باشه. 

من هم وبلاگت رو دیدم و داخل کانال تلگرامت عضو شدم. همچنین از خوندن پست [بعد از دو ماه دوری از اینستاگرام - ویرگول](https://virgool.io/@mohsenkhavari/%D8%A8%D8%B9%D8%AF-%D8%A7%D8%B2-%DB%8C%DA%A9-%D9%85%D8%A7%D9%87-%D8%AF%D9%88%D8%B1%DB%8C-%D8%A7%D8%B2-%D8%A7%DB%8C%D9%86%D8%B3%D8%AA%D8%A7%DA%AF%D8%B1%D8%A7%D9%85-iqpdwq4shhpb)لذت بردم. من هم از دوری از این شبکه اجتماعی سمی (!) راضیم و الان میشه گفت ۵ سال (یا حتی بیشتر هست) که ندارمش. البته چند ماه یک‌بار سر می‌زنم ولی کاملاً کنترل‌شده است و اگر نبود هم اصلاً برام مهم نیست. 

بعضی شبکه‌های اجتماعی محیط‌شون سمیه. بعضی‌ها اصلاً اینطور طراحی شدند. توییتر هم برای من همچین حسی رو تداعی می‌کنه. 

راجع به وبلاگ به نظرم فعلاً برای یکی مثل تو که تخصصش پزشکی هست و نمی‌خواد درگیر پیچیدگی‌های فنی بشه، راحت‌ترین کار (و شاید بهترین) نگه‌داشتن همین اکانت ویرگولی هست که داری. 

اگر برنامه‌نویس بودی، قطعاً بهت پیشنهاد می‌دادم که وب‌سایت شخصی راه بیندازی. بالاخره کمترین انتظاری که میره، اینه که حداقل یک وب‌سایت شخصی داشته باشه که رزومه‌اش و وبلاگش رو هاست کنه و اگر هم پیچیدگی فنی‌ای داره، خودش رفعش کنه. مثلاً کار کردن با Git و داشتن کلید SSH و Debugکردن یک خطای فنی از الزامات یک برنامه‌نویسه. 

تو بعضی پوزیشن‌های برنامه‌نویسی داشتن وب‌سایت شخصی حتی تو استخدام‌شدن یا نشدن تأثیر داره. حتی بعضاً تکنولوژی‌ای که برای وب‌سایت استفاده شده براشون اهمیت داره. 

اما این‌ مورد برای یک پزشک چندان معنایی نداره. به شخصه هم ویرگول و هم از وردپرس بدم میام و حاضر نیستم باهاشون کار کنم؛ ولی فکر می‌کنم اگر نخوای درگیر پیچیدگی‌های فنی بشی، همین دو تا گزینه معقول هستند.

وردپرس می‌تونه برات جذاب باشه به شرطی که بخوای دامنه و هاستینگ اختصاصی داشته باشی (که اون هم دردسرهای خاص خودش رو داره) و ویرگول رو هم که خودت می‌دونی چطوریه. تصمیم با خودت.

**Comment by HashemAlavi** [comment]
سلام🙌

یادمه ماه قبل که گزارش ماهانه‌ت رو خوندم می‌خواستم کامنت بذارم. بعد دیدم در مورد دو سه تا چیز می‌خوام بنویسم که گفتم الآن تمرکز ندارم. بذار فردا پس فردا برمی‌گردم مینویسم. ولی خب کلاً یادم رفت تا امروز که گزارش جدید این ماه رو منتشر کردی.

ولی خب حالا که چیز خاصی برای گفتن ندارم گفتم لااقل یک اعلام حضوری کنم و در مورد شب‌های برره بگم که سریال باحال و خنده داری بود. من خیلی حال کردم باهاش (فکر کنم یکی دو سال پیش نشستم همشو یبار از اول دیدم!).

راستی یک سوالی هم تو ذهنم بود در مورد اینکه اگه یک نفر مثل من بخواد یک سری مفاهیم پایه‌ای در مورد هوش مصنوعی بدونه (بیشتر من باب اطلاعات عمومی که میخوام بدونم چخبره) باید چیکار کنم؟
مثلاً در مورد NLP الآن یک سری سوال دارم. در مورد اینکه LLM و اینچیزا چیه چجوری داره کار می‌کنه؟ دنبال کد زدن و اینچیزا هم نیستم. میشه لطفاً در این مورد یکم راهنماییم کنی؟

**Reply by pourmand1376** [reply]
سلام هاشم جان. خوشحال شدم پیامت رو دیدم. 

آره هاشم. سریال جالبی هست. از اون کمدی‌هایی هست که آدم هر وقت ببینه، مصداق‌هاش رو اطرافش می‌بینه و این از تیزهوشی مهران مدیری هست. 

راجع به چیزایی که گفتی، فعلاً جایی رو نمی‌شناسم که تو سطحی که تو می‌خوای مطالبی که میگی رو ارائه بده. اصلاً به همین خاطر می‌خواستم پادکست بدم و قرار بود یکی دو قسمت دیگه که از معرفی کلیت هوش مصنوعی گذشتم، راجع به LLMها صحبت کنم. اتفاقاً خیلی هم برام جذاب هستند. 

ولی چند وقت گذشته زیاد وقت نداشتم و اصلاً فرصت نشد. از طرفی از روندی که قسمت‌های قبلی رو جلو بردم، راضی نیستم. به نظرم حوصله سربر و «آکادمیک» شده. یه پادکست اصلاً نباید وارد این جزئیات بشه. 

البته اگر می‌خوای راجع به این مطالب در زبان انگلیسی بدونی، خیلی‌ها به زبان ساده صحبت کردند. کافیه تو یوتیوب سرچ کنی و ویدئوهای خیلی جذابی راجع بهش میاد؛ ولی تو فارسی من ندیدم.

**Reply by HashemAlavi** [reply]
خیلی ممنون از پاسخ قشنگت:) ببخشید من یکم دیر دارم جواب میدم. دو روزی میشه که لپتاپ رو فرستاده بودم برا تعمیر. امروز تازه رسید به دستم.

آره یوتیوب به ذهنم نرسیده بود (خیلی چیزها توش سرچ می‌کنم ولی این مورد رو خیلی نگشتم). به اینم سر میزنم.

راستی من امروز هم 4 قسمتی که از ایستگاه هوش مصنوعی منتشر کردی رو تو کست باکس گوش کردم. خیلی خوب بود هم از جهت اینکه محتوا ساختار داشت و خیلی از این شاخه به اون شاخه نبود. و هم اینکه مثال‌هایی که زدی رو دوست داشتم (چون از حوزه‌های مختلف مثال آورده بودی و متنوع بود؛ و اینکه من حس بدی نگرفتم و به‌نظرم نیومد که خیلی وارد جزئیات بیهوده شده باشی. تایم پادکست هم واقعاً معقول بود).

خلاصه که دمت گرم. منتظر قسمت‌های بعدی هستم🙂
